Quickstart: Switch to directly downloading release artifacts (#524)
precice/tutorials: quickstart/README.md

    sudo apt install build-essential pkg-config cmake git
    ```

4. Download and install the [OpenFOAM-preCICE adapter](https://precice.org/adapter-openfoam-get.html) from the [latest release](https://github.com/precice/openfoam-adapter/releases/latest), unpack it, and build the adapter:

    ```bash

>      wget https://github.com/precice/openfoam-adapter/archive/refs/tags/v1.3.0.tar.gz

Reviewer: If you would want, we could also make this version agnostic and just download the latest version:
```suggestion
LATEST_RELEASE_JSON=$(curl -s https://api.github.com/repos/precice/openfoam-adapter/releases/latest)

# Use jq to parse JSON and get the tarball URL, we could potentially use grep here as well
TARBALL_URL=$(echo $LATEST_RELEASE_JSON | jq -r ".tarball_url")
wget $TARBALL_URL
```

Author: But then we are back to square one: We make a preCICE release that does not work with the latest tutorial release, and we have the same issue.

Reviewer: We can only refer to v1.3.0 here because it's out now. If it wouldn't be out, we would face the incompatibility anyway, right?

Author: Yes, but then we would only update this page once all releases of the required components are out.

Reviewer: I think the minor releases, which would require an update here are more frequent than the major releases you have in mind. Still, I don't have a strong opinion, so you could also just stick to fixed versions, if you think it is more suitable.

Author: Good point about the minor releases. If you want to try and extract this from the strings curl returns, go ahead. I would be very happy if we can merge this today.

Author: Let's keep the pinned versions for now, we can attempt a better solution later.

     tar -xzf v1.3.0.tar.gz 
     cd openfoam-adapter-1.3.0/
     ./Allwmake
     cd ..
    ```

5. Get the [latest release of the tutorials](https://github.com/precice/tutorials/releases/latest), or get the latest `master` branch of the repository and navigate to the quickstart tutorial:

    ```bash
    wget https://github.com/precice/tutorials/releases/tag/v202404.0
    tar -xzf v202404.0.tar.gz
    cd tutorials/quickstart
    ```

> If you prefer to easily try everything in an isolated environment, you may prefer using our [demo virtual machine](https://precice.org/installation-vm.html). If you are looking for previous versions, check out the [preCICE Distribution](https://precice.org/installation-distribution.html).

Author: I added again a mention to the preCICE Distribution, this time only as a hint for older versions.

I am not sure who would look for older versions, but I feel this is a good place to tell users "hey! you are looking for something you found in a paper? Maybe look here, instead before you start guessing which versions you need".

Reviewer: ```suggestion
If you prefer to easily try everything in an isolated environment, you may prefer using our [demo virtual machine](https://precice.org/installation-vm.html). If you are looking for previous versions or face compatibility issues, check out [preCICE Distribution](https://precice.org/installation-distribution.html).
```

Reviewer: The intrinsic problem here is also, that people typically don't recognize compatibility issues as such. If someone couldn't locate a `SolverInterface.hpp`, most likely this doesn't sound as if you would use the wrong version, right? It's rather a bogus installation..

Author: Isn't this then the same as the note I originally had and showed a fragile system? I agree we should not give that impression, but I am trying to find a subtle way to inform the users about such a potential (but maybe unlikely) issue.

Reviewer: Hm, I guess it's more about the wording. Here, it says: if you face problems, look in the distri. In the former version it was: All versions which are known to be compatible are in our distribution. To me, there is a slight difference in the underlying assumption.

Author: Convincing enough for me, thanks! (and added back a missing `the`)

    sudo apt install build-essential pkg-config cmake git
    ```

> 4. Download and install the [OpenFOAM-preCICE adapter](https://precice.org/adapter-openfoam-get.html) from the [latest release](https://github.com/precice/openfoam-adapter/releases/latest), or get the latest `master` branch of the repository:

Reviewer: Didn't we want users to use the release archives? Also to accommodate for the different OpenFOAM versions available.

Author: Yes, but how would you write that in one command? We would always need to update the version number manually.

Author: @davidscn we can still merge this part, so that at least we hint towards releases.

We could add the versions as variables, but these would only render on the website, and we would need to remember to update them whenever we want users to use something new.

Reviewer: ```suggestion
4. Download and install the [OpenFOAM-preCICE adapter](https://precice.org/adapter-openfoam-get.html) from the [latest release](https://github.com/precice/openfoam-adapter/releases/latest)´
```
Just like this? In the release, we anyway distinguish the OpenFOAM version?!

Author: Yes, good point, we already specify the OpenFOAM version here, so let's keep everything fail-safe: we ask users to install specific versions. Even if we don't immediately update everything, these should still work.

If you prefer to easily try everything in an isolated environment, you may prefer using our [demo virtual machine](https://precice.org/installation-vm.html).

{% note %}
We are assuming here that the latest `master` branches refer to the latest releases, and that they all work together.
This might not be the case in transition periods after a major preCICE release (e.g., v2 to v3).
Versions that are guaranteed to work together are described in the [preCICE Distribution](https://precice.org/installation-distribution.html).
{% endnote %}

> 

Reviewer: I'm not too sure about this Note. The transition period essentially ends with the release, where you want to include it. To me, it confuses more than helps. It gives overall the impression of a fragile system, where you have to write down which versions and packages are compatible with each other. That's not really the case for preCICE and its components.

Author: I see your point, and I was mainly aiming for the v3->v4 transition. I can remove this, but then we can just close this PR.

Author: Removed
